atd-toolbox/data_model/python/populate_edits_sql/units.sql

```diff
@@ -0,0 +1,106 @@
+set client_min_messages to warning;
+
+
+-- todo: double check if CRIS actually has the right data for this unit
+update atd_txdot_units set unit_desc_id = 8 where unit_desc_id = 94;
+
+create materialized view unit_diffs as with joined_units as (
+    select
+        unit_unified.id,
+        unit_unified.unit_nbr as unit_nbr,
+        unit_unified.crash_pk as crash_pk,
+        unit_unified.cris_crash_id as cris_crash_id,
+        unit_edit.travel_direction as veh_trvl_dir_id_edit,
+        unit_edit.movement_id as movement_id_edit,
+        unit_edit.unit_desc_id as unit_desc_id_edit,
+        unit_edit.veh_body_styl_id as veh_body_styl_id_edit,
+        unit_unified.veh_trvl_dir_id as veh_trvl_dir_id_unified,
+        unit_unified.movement_id as movement_id_unified,
+        unit_unified.unit_desc_id as unit_desc_id_unified,
+        unit_unified.veh_body_styl_id as veh_body_styl_id_unified
+    from
+        units as unit_unified
+    left join atd_txdot_units as unit_edit on
+        unit_unified.cris_crash_id = unit_edit.crash_id
+        and unit_unified.unit_nbr = unit_edit.unit_nbr
+),
+
+computed_diffs as (
+    select
+        id,
+        unit_nbr,
+        cris_crash_id,
+        crash_pk,
+        case
+            when
+                veh_trvl_dir_id_edit is distinct from veh_trvl_dir_id_unified
+                and veh_trvl_dir_id_edit is not null
+                then
+                    veh_trvl_dir_id_edit
+        end as veh_trvl_dir_id,
+        case
+            when
+                movement_id_edit is distinct from movement_id_unified
+                and movement_id_edit is not null
+                -- importantly, this ignores where movement_id = 0 (UNKNOWN) in the old schema
+                -- since it's a default value we don't intend to carry forward
```

Comment:
:+1:

Reply:
`movement_id` is actually a fully custom VZ column. it doesn't exist in CRIS at all

Reply:
ahh right !